dist/scripts/api-console.js

```diff
@@ -756,12 +756,20 @@
           $scope.listArray = $scope.list;
         }
 
+        var getArrayTypes = function(arrayType) {
+          if (arrayType.items.type || Array.isArray(arrayType.items.type)) {
```

Comment:
Will always arrayType have defined its `items` attribute? I'm wondering if there's a chance of having a runtime error if `arrayType.items` is `undefined`.